### DISCUSSION TITLE
Seems Invalidate when using image2image in batches from a folder

### DISCUSSION THREAD

**Discussion by willliamahician** [adieyal/sd-dynamic-prompts#279] [Q&A] [answered]
It looks invalid when batch drawing images from a folder. It works very well when I use wildcards to batch generate text2text and image2image images (each result is random), but once I batch generate from a folder to other, the plug-in will randomly select a keyword and apply it to all images (each result is the same)

**Comment by adieyal** [comment, marked as answer]
Makes sense - I haven't tested with batch images - will investigate.

**Reply by adieyal** [reply]
I've created an issue here: https://github.com/adieyal/sd-dynamic-prompts/issues/281
